src/widgets/section/SectionList.tsx

```diff
@@ -41,12 +48,105 @@ export default class SectionList extends WidgetBase<SectionListParameters> {
 		);
 	}
 
+	private _getContent() {
+		const { section, listIndex } = this.properties;
+		if (listIndex) {
+			return this.meta(Block).run(compiler)(section, listIndex, {
+				locale: i18n.locale
+			})
+		}
+
+		return null;
+	}
+
+	private _parseList(subsection: string, list?: DNode | DNode[]): DNode[] {
```

Comment:
might be worth you writing a tiny node recursing function that takes a predicate and a replacer function rather than all in here